yarn.lock

```diff
@@ -36073,7 +36068,6 @@ __metadata:
     "@automattic/calypso-typescript-config": "workspace:^"
     debug: "npm:^4.4.0"
     postcss: "npm:^8.5.3"
-    uuid: "npm:^9.0.1"
```

Comment:
Ah well, it would be nice to do this replacement in Gutenberg at some point, too.

Reply:
Yes 😿